android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -561,24 +550,43 @@ public boolean isMapAttached()
 
   private void initNavigationButtons()
   {
-    View frame = findViewById(R.id.navigation_buttons);
-    if (frame == null)
-      return;
 
-    View zoomIn = frame.findViewById(R.id.nav_zoom_in);
-    zoomIn.setOnClickListener(this);
-    View zoomOut = frame.findViewById(R.id.nav_zoom_out);
-    zoomOut.setOnClickListener(this);
-    View myPosition = frame.findViewById(R.id.my_position);
-    mNavMyPosition = new MyPositionButton(myPosition, mOnMyPositionClickListener);
+    View frame = findViewById(R.id.navigation_buttons);
+    if (frame != null)
```

Comment:
When it can be null?

Reply:
It is a leftover from the original code but you are right it should not be possible (and if it is there better be a crash than a silent error)